src/isar/services/readers/mission_reader.py

```diff
@@ -68,12 +68,10 @@ def get_predefined_missions(self) -> Optional[dict]:
             return missions
         except Exception as e:
             logger.error(f"Error in reading of files {e}")
-            return None
+            raise MissionReaderError
```

Comment:
Should you not remove this as well? Either have log statement in exception or remove the whole except? The same question other places.